docs/styles.scss

```diff
@@ -37,13 +35,56 @@ body {
 
     .tab-bar-horizontal { margin-bottom: 25px; }
 
+    .width-overflow{ overflow-x: auto; }
+
     code {
         max-width: 870px;
     }
 }
 
+.full-height > .vertical-container { min-height: calc( 100vh - 53px ); }
```

Comment:
I've had mixed results with using `vh`. What was the difficulty presented with using `%` instead?